pkg/sql/executor_ir.go

```diff
@@ -61,11 +61,8 @@ func RunSQLProgram(sqlProgram string, modelDir string, session *pb.Session) *pip
 		}
 		err = runSQLProgram(wr, sqlProgram, db, modelDir, session)
 		if err != nil {
-			err = wr.Write(fmt.Errorf("runSQLProgram error: %v", err))
-			if err != pipe.ErrClosedPipe {
-				if err := wr.Write(err); err != nil {
-					log.GetDefaultLogger().Errorf("runSQLProgram error(piping): %v", err)
-				}
+			if e := wr.Write(fmt.Errorf("runSQLProgram error: %v", err)); e != nil {
+				log.GetDefaultLogger().Errorf("runSQLProgram error(piping): %v", err)
```

Comment:
This change may cause the client (like in Jupyter Notebook) will not know the error message if the Python program fails?

Reply:
May not, L64 tried to write `fmt.Error` to the pipe.